src/promnesia/sources/pocket.py

```diff
@@ -7,22 +7,43 @@
 def index() -> Results:
     from . import hpi
     from my.pocket import articles
+
     # TODO use docstring from my. module? E.g. describing which pocket format is expected
 
     for a in articles():
-        loc = Loc.make(title='pocket', href=a.pocket_link)
+        title = a.json.get('resolved_title', 'pocket')
+        loc = Loc.make(title=title, href=a.pocket_link)
+        # Add a reverse locator so that the Promnesia browser extension shows a
+        # link on the Pocket page back to the original URL.
+        loc_rev = Loc.make(title=title, href=a.url)
         hls = a.highlights
         if len(hls) == 0:
             yield Visit(
                 url=a.url,
                 dt=a.added,
-                context=None,
+                context=a.json.get('excerpt', None),
                 locator=loc,
             )
+            # Add a visit with the Pocket URL so that we can see that we have
+            # visited before in Promnesia browser extension.
+            yield Visit(
+                url=a.pocket_link,
+                dt=a.added,
+                context=a.json.get('excerpt', None),
+                locator=loc_rev,
+            )
         for hl in hls:
             yield Visit(
                 url=a.url,
                 dt=hl.created,
                 context=hl.text,
                 locator=loc,
             )
+            # Add a visit with the Pocket URL so that we can see that we have
+            # visited before in Promnesia browser extension.
+            yield Visit(
```

Comment:
same as above -- not sure there is much point adding all these visits for each highlight when you're on Pocket website anyway and can see them in Pocket interface? But again don't think it's a big deal either way so curious to know what you think